sites/perioimplantadvisory/server/templates/subscribe/index.marko

```diff
@@ -30,10 +30,8 @@ $ const newsletterDescription = 'Stay up-to-date on industry news and events, ne
           <p>${newsletterDescription}</p>
           <p><strong><a href="/subscribe/email">Subscribe / Manage Preferences</a></strong></p>
           <hr>
-          <h5>Magazines</h5>
         </div>
         <div class="page-wrapper__body">
```

Comment:
You can safely remove the wrapping div since the contents have been removed.

Reply:
done